sdk/storage/azure-storage-datamovement/inc/azure/storage/datamovement/storage_transfer_manager.hpp

```diff
@@ -16,16 +20,33 @@ namespace Azure { namespace Storage {
     explicit StorageTransferManager(const StorageTransferManagerOptions& options);
     virtual ~StorageTransferManager() = 0;
 
-    void PauseJob(const std::string& jobId);
-    void PauseAllJobs();
-    void ResumeJob(const std::string& jobId);
-    void ResumeAllJobs();
     void CancelJob(const std::string& jobId);
     void CancelAllJobs();
```

Comment:
We may need to discuss about concept of cancel/pause.. 
Do we really need to provide a method to cancel all?

Reply:
Let's discuss this in weekly meeting. I can remove this if we agree it's not necessary.